src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandLineHelpViewer.cs

```diff
@@ -0,0 +1,120 @@
+using Microsoft.DotNet.Cli.CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    internal static class CommandLineHelpViewer
+    {
+        private static int columnGutterWidth = 3;
```

Comment:
consider PASCAL naming for constants\static fields.
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/d53b55ey(v=vs.71)